### ISSUE TITLE
Update guides to take advantage of existing codestarts

### ISSUE THREAD

**Issue by ia3andy** [quarkusio/quarkus#14444]
**Description**

Those guides present extensions which already have a codestart and which could be adapted to use it:
- [ ] https://quarkus.io/guides/picocli
- [ ] https://quarkus.io/guides/rest-json
- [ ] https://quarkus.io/guides/logging#json-logging
- [ ] https://quarkus.io/guides/amazon-lambda
- [ ] https://quarkus.io/guides/azure-functions-http
- [ ] https://quarkus.io/guides/funqy-amazon-lambda
- [ ] https://quarkus.io/guides/funqy-http
- [ ] https://quarkus.io/guides/funqy-knative-events
- [ ] https://quarkus.io/guides/gcp-functions
- [ ] https://quarkus.io/guides/gcp-functions-http
- [ ] https://quarkus.io/guides/qute

cc @maxandersen

**Comment by gsmet** [comment]
We need to be extremely careful about this: the guide still needs to be explanatory and to contain the important information for the user.

Having the project all created by codestarts won't help to understand what's going on.

**Comment by ia3andy** [comment]
> We need to be extremely careful about this: the guide still needs to be explanatory and to contain the important information for the user.
> 
> Having the project all created by codestarts won't help to understand what's going on.

Yes I agree, as a guideline for our guide, we could advise to use `-noExamples` to do the tuto and remove it (or see the quickstart) to directly have the solution.

**Comment by ia3andy** [comment]
In 2.0 codestarts are the beginning of guides, so we can now consider using them.
